indico_toolkit/association/positioning.py

```diff
@@ -103,12 +103,68 @@ def get_min_distance(
             min_distance += (page_height * page_difference)
         return min_distance
 
+    @staticmethod
+    def get_vertical_min_distance(
+        above_pos: dict, below_pos: dict, page_height: int = None
+    ) -> float:
+        """
+        Get the vertical minimum distance between two bounding boxes
+        Args:
+            above_pos (dict): the position expected to be above
+            below_pos (dict): to position expected to be below
+            page_height (int, optional): If you want to measure distances across pages, set the OCR page height
+                                         otherwise locations on separate pages will raise an exception.
+                                         Defaults to None.
+
+        Returns:
+            float: minimum distance
+        """
+        add_page_height = False
```

Comment:
if we just set add_page_height to False here, think we can remove all of the other logic that includes it right?

Reply:
oh whoops ignore me